src/views/EventInitiation.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="page-container">
+    <NavBtns
+      :topType="'share-dark'"
+      :botType="currStep === 0 ? 'next-only' : 'default'"
+      @next="nextStep()"
+      @back="prevStep()"
+    />
+
+    <!-- step 1 -->
+    <div class="step-1 step" v-if="currStep === 0">
+      <img src="../assets/icons/event-initiation/hourglass-icon.svg" alt class="step-icon" />
+      <h1 class="step-title">Date / Time</h1>
```

Comment:
Might want to make this `user-select: none` so that its not confusing of what can/cannot be clicked.

Reply:
wait can u elaborate on what this will achieve

Reply:
Just small change that will remove text-highlighting, from ux perspective it might just guide users better on what to click on